packages/fxa-content-server/server/lib/csp/blocking.js

```diff
@@ -34,7 +34,7 @@ module.exports = function (config) {
     config.get('pairing.server_base_uri')
   );
   const PAIRING_SERVER_HTTP = PAIRING_SERVER_WEBSOCKET.replace(/^ws/, 'http');
-  const SENTRY_SERVER = 'https://sentry.prod.mozaws.net';
+  const SENTRY_SERVER = 'https://*.sentry.io';
```

Comment:
What all does the `*` cover? If there's only a few of these it might feel a bit safer to name them explicitly.